manifests/params.pp

```diff
@@ -48,7 +48,23 @@
 
   case $::osfamily {
     'Debian': {
-      $service_provider = 'upstart'
+
+      if $install_method == 'archive' {
+        $manage_unzip = true
+      }
+
+      case $::lsbdistcodename {
+        /(jessie|stretch|sid|xenial|yakketi|zesty)/: {
```

Comment:
No problem, I can look at officially supporting those distros at a later date but this code will help that process.